src/components/abc-input.js

```diff
@@ -107,7 +108,7 @@ function AbcInput({
 
   const renderToolsButton = () => (
     <Dropdown
-      placement="top"
+      placement="topLeft"
```

Comment:
This change was necessary, because the Antd dropdown was in a state of constant flickering when on a narrow screen and reaching the right page margin.